diff --git a/emulator/emulator.py b/emulator/emulator.py
--- a/emulator/emulator.py
+++ b/emulator/emulator.py
@@ -389,6 +389,8 @@
                     self.memory[0x3E] = 0x0A
                 elif event.key == pygame.K_TAB:
                     self.memory[0x3E] = 0x09
+                elif event.key == pygame.K_ESCAPE:
+                    self.memory[0x3E] = 0x1B
                 if event.key == pygame.K_BACKSPACE:
                     self.memory[0x3E] = 0x08
         #
